### DISCUSSION TITLE
er图数据如何单对单单向连接

### DISCUSSION THREAD

**Discussion by 15643701048a** [antvis/X6#2211] [Q&A]
![image](https://user-images.githubusercontent.com/77105350/173001509-f1738a55-78fd-4802-9da9-5ce2f17f05c3.png)
![image](https://user-images.githubusercontent.com/77105350/173001621-7d1291b4-17a4-4ebe-8c81-99bcd8a4e205.png)
![image](https://user-images.githubusercontent.com/77105350/173001669-c7956721-ee63-4f43-9edc-70e9e0b5615e.png)

er图数据是由链接桩显示的，目前可以是多对多双向连接的，请问port如何单对单单向连接

**Comment by NewByVector** [comment]
可以限制 port 的连接数量，希望这个 [demo](https://codesandbox.io/s/xian-zhi-lian-jie-bian-shu-liang-ytk2rd) 能帮到你。

**Reply by 15643701048a** [reply]
按照官网的例子validateMagnet获取的magnet.getAttribute('port')为null
![image](https://user-images.githubusercontent.com/77105350/174934536-69665f61-41c5-4f58-bb4b-0821d8011691.png)
是不是因为list的原因

**Reply by 15643701048a** [reply]
我知道了，用它的父节点就获取到了
